help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,213 @@
+![Add new mapping](../../assets/catalog/crm/microsoft-dynamics-365/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/microsoft-dynamics-365/source-mapping.png)
+
+
+1. In the "[!UICONTROL Select target field]" window, select the target field 
+    * **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
+    ![Target mapping using contactId](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-contactid.png)
+
+    * Add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+        |XDM Profile Schema|[!DNL Outreach] Instance| Mandatory|
+        |---|---|---|
+        |`contactId`|`contactId`| Yes |
+
+    * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1#entity-properties) for a comprehensive list of supported attributes.
+    ![Target mapping using LastName](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-lastname.png)
+
+    * For instance, depending on the values you want to update, add the following mapping between your XDM profile schema and your [!DNL Outreach] instance:
+        |XDM Profile Schema|[!DNL Dynamics 365] Instance|
+        |---|---|
+        |`person.name.firstName`|`FirstName`|
+        |`person.name.lastName`|`LastName`|
+        |`personalEmail.address`|`Email`|
+
+1. An example using these mappings is shown below:
+![Target mapping](../../assets/catalog/crm/microsoft-dynamics-365/mappings.png)
+
+> [!IMPORTANT]
+>
+> When mapping to a Dynamics 365 [date or timestamp](https://docs.microsoft.com/en-us/power-apps/developer/data-platform/webapi/reference/timestampdatemapping?view=dataverse-latest), the corresponding attribute value in the dataset should not be empty else during execution, else you will encounter [errors](#unknown-errors).
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you must manually map Platform segments to the custom field attribute in [!DNL Dynamics 365].
+
+To do this, select each segment, then enter the corresponding custom field attribute from [!DNL Dynamics 365] in the **[!UICONTROL Mapping ID]** field.
+
+> [!IMPORTANT]
+>
+> The value used for the Mapping ID should exactly match the name of the custom field attribute created within [!DNL Dynamics 365].
```

Comment:
Let's add a link to the Dynamics docs where they explain where to find the custom field name.